content/source/docs/enterprise/private/reliability-availability.html.md

```diff
@@ -226,30 +187,48 @@ The procedure here would be as follows:
   instance. The exact mechanism used is specific to the virtualization
   environment that the product runs in.
 
+**Restore**
+
 - Configure the virtualization to run the new snapshot and restart a new
   instance should it stop. In AWS, this is done using an Autoscaling Group.
   The environment can monitor the availability of port 443 for instance health
   as well.
 
-The new instance now has the software as well as configuration on it by default
-and can access the data stored on the mounted disk. Using these steps, the MTTR
-of a lost instance is almost as fast as the virtualization environment can start
-a new instance, typically less than a minute.
+- The new instance now has the software as well as configuration on it by default
+  and can access the data stored on the mounted disk. Using these steps, the MTTR
+  of a lost instance is almost as fast as the virtualization environment can start
+  a new instance, typically less than a minute.
 
 ~> **NOTE:** Because the software is being restored from a snapshot, it's
 important that this process be repeated when the Terraform Enterprise software
 is updated so that any restored instance due to loss has the newest version.
 This is important because the data stored on the mounted disk is versioned
 along with the software.
 
-#### External Services
+### External Services
+
+In the External Services mode of the Installer Architecture, the
+**Application Layer** and **Coordination Layer** execute on a Linux instance,
+but the **Storage Layer** is configured to use external services in the form an
+a PostgreSQL server, an S3-compatible Blob Storage, and an
+[external Vault](./vault.html).
```

Comment:
@evanphx flagging this change (referencing the external Vault guide) for you to review specifically

Reply:
That's fine but it should be spelled out that using an external Vault is an optional configuration, not something that is used by default with external services mode.